[Master] - Fixed Configuring Account Disabling Doc Issues (#2912)
wso2/docs-is: en/docs/learn/configuring-account-disabling.md

## Scenario

**Pickup** is a cab company that has many employees who use different credentials to sign in to different internal enterprise applications. **Sam** is an administrator at Pickup. **Larry** is an employee who had served Pickup for 7 years and is entitled for sabbatical leave. As Larry is going on sabbatical leave for 2 years, Sam wants to disable Larry's account.  

Let's learn how Sam can disable Larry's user account! 

## Set up

Follow the steps below to configure account disabling in WSO2 Identity Server.

1.	Open the `deployment.toml` file in the `<IS_HOME>/repository/conf` directory.

	1.	Check whether the following listener configs are in place.

		```toml
		[event.default_listener.identity_mgt]
		priority= "50"
		enable = false
		[event.default_listener.governance_identity_mgt]
		priority= "95"
		enable = true
		```

> 	2.	To configure the email server to send emails that informs about the disabled account to the user, add the following configurations.

Reviewer: ```suggestion
	2.	To configure the email server to send emails that informs about the disabled account to the user, add the following configurations:
```

		-	**from_address**: This is the email address from which the confirmation email will be sent.
		-	**username**: This is the username of the given email address.
		-	**password**: This is the password of the given email address. 

		```toml tab="Format"
		[output_adapter.email]
		from_address= ""
		username= ""
		password= ""
		hostname= "smtp.gmail.com"
		port= 587
		enable_start_tls= true
		enable_authentication= true
		```

		```toml tab="Sample"
		[output_adapter.email]
		from_address= "wso2iamtest@gmail.com"
		username= "wso2iamtest"
		password= "Wso2@iam70"
		hostname= "smtp.gmail.com"
		port= 587
		enable_start_tls= true
		enable_authentication= true
		```

		!!! warning "If you are using a Google email account"

> 			Google has restricted third-party applications and less secure applications from sending emails by default. As WSO2 Identity Server acts as a third-party application when sending emails for password entry, follow the steps below to enable your Google email account to provide access to third-party applications.

Reviewer: ```suggestion
			Google has restricted third-party applications and less-secure applications from sending emails by default. As WSO2 Identity Server acts as a third-party application when sending emails for password entry, follow the steps below to enable your Google email account to provide access to third-party applications.
```

			1.	Access [https://myaccount.google.com/security](https://myaccount.google.com/security).

> 			2.	Under **Signing in to Google** section, turn off the **2-step Verification** option.

Reviewer: ```suggestion
			2.	In the **Signing in to Google** section, turn off the **2-step Verification** option.
```

> 			3.  Enable **Less secure app access** in Google Account **Security** section.

Reviewer: ```suggestion
			3.  Enable **Less secure app access** in the Google Account's **Security** section.
```

		<img src="../../assets/img/learn/account-disabled-email.png" alt="Account Disabled email" width="500" style="border:1px solid grey"> 

	5.	Access the WSO2 Identity Server My Account at `https://localhost:9443/myaccount`.

		<img src="../../assets/img/learn/userportal-login-screen.png" alt="Sign In form" width="400" style="border:1px
		 solid grey"> 	

	6.	Try logging in with Larry's credentials. Note that an error message appears.

4. To revoke the account disabling:

	1.	Click **User Profile** option of Larry.

> 	2.	Unselect the **Account Disabled** check box.

Reviewer: ```suggestion
	2.	Clear the **Account Disabled** checkbox.
```

	3.	Click **Update**.

	4.	An email that informs about the enabled account is sent to the given email address.

		<img src="../../assets/img/learn/account-enabled-email.png" alt="Account Enabled email" width="500" style="border:1px solid grey"> 		

> 	5.	Trying logging in to the WSO2 Identity Server User Portal with Larry's credentials. The WSO2 Identity Server

Reviewer: ```suggestion
	5.	Try logging in to the WSO2 Identity Server's My Account with Larry's credentials. The WSO2 Identity Server
```
